R/standalone-types-check.R

```diff
@@ -470,4 +473,28 @@ check_character <- function(x,
   )
 }
 
+check_logical <- function(x,
+                          ...,
+                          allow_null = FALSE,
```

Comment:
I wonder if the vector variants should also get the same parameters as the scalar variants. Not for this PR, just a general thought.